src/lib/http.coffee

```diff
@@ -21,11 +23,30 @@ module.exports = (dnschain) ->
             @log = gNewLogger 'HTTP'
             @log.debug "Loading HTTPServer..."
             @rateLimiting = gConf.get 'rateLimiting:http'
+            app = express()
+            router = express.Router()
+
+            router.route(/\/v1\/(\w+)\/(\w+)(\/([^\/]+)(\/(\w+)(\.(\w+))?)?)?/)
+            .get (req, res) =>
+                [chain, resource, _, property, _, operation, _, fmt] = req.params
+                @log.debug gLineInfo('GET v1 API'),
+                    {chain:chain, resource:resource, property:property, operation:operation, fmt:fmt}
+                if chain is 'resolver'
+                    # TODO do resolver stuff
+                    res.end()
```

Comment:
I would do this via a separate `.get` that's mounted at `"/resolver"`, i.e. add another `.get("/resolver/:resource", ...` or something like that.